### PR TITLE
Add GetClassMembers endpoint

### PR DESCRIPTION
**Pull request by joshSi**

### CODE REVIEW

**Comment by joshSi** [comment]
Oh sorry meant to make a draft

**Comment by krashanoff** [comment]
@joshSi you can reopen this pull request and then just edit it to mark it as a draft!

File: db/class.go [hunk 1]
```diff
@@ -421,3 +421,65 @@ func (d *DB) DeleteClass(c echo.Context) error {
...
+func (d *DB) GetClassMembers(c echo.Context) error {
+	var (
+		req struct {
+			CID string `json:"cid"`
+			UID string `json:"uid"`
+		}
+
+		res map[string]User
+		err error
+	)
```
Comment (krashanoff) [inline db/class.go:436]: You can probably get by removing the extra variable declarations here then use the short assignment statement, as you do later in the function.
```suggestion
	var req struct {
		CID string `json:"cid"`
		UID string `json:"uid"`
	}
```

File: db/class.go [hunk 2]
```diff
@@ -421,3 +421,65 @@ func (d *DB) DeleteClass(c echo.Context) error {
...
+			isIn = true
+			break
+		}
+	}
+
+	if !isIn {
+		return c.String(http.StatusNotFound, "given user not in class")
+	}
+
+	res = make(map[string]User)
```
Comment (krashanoff) [inline db/class.go:466]: Related to the previous review comment:

```suggestion
	res := make(map[string]User)
```

**Review by krashanoff — CHANGES_REQUESTED** [review]
Just a quick review. This looks good following a coding style change. Thanks for the work Josh!

**Review by krashanoff — APPROVED** [review]
We're seriously going to need to fix our tests. Aside from this, the changes look good. Thanks for your work Josh!

**Comment by joshSi** [comment]
Closes #46

**Comment by joshSi** [comment]
Closed #36

**Pull request merged**